otlp/src/Proto/Opentelemetry/Proto/Collector/Logs/V1/LogsService.hs

```diff
@@ -0,0 +1,641 @@
+{- HLINT ignore -}
```

Comment:
I suggest telling github that these files are generated: https://docs.github.com/en/repositories/working-with-files/managing-files/customizing-how-changed-files-appear-on-github

Reply:
done